app/components/reusables/icon-button.hbs

```diff
@@ -0,0 +1,8 @@
+<button
+  class='icon-button {{@class}}'
+  type='button'
+  {{on 'click' (fn @onClick @id)}}
+  data-test-icon-button={{@id}}
+>
+  <IconifyIcon data-test-icon={{@id}} @icon={{@icon}} />
```

Comment:
Is I am right??
This is component and you not created this component function ??

Reply:
Yeah this is component and functions are passed into it as a props